edgedb/con_utils.py

```diff
@@ -861,6 +865,12 @@ def _parse_cloud_instance_name_into_config(
     org_slug: str,
     instance_name: str,
 ):
+    label = f"{instance_name}--{org_slug}"
+    if len(label) > 63:
+        raise ValueError(
```

Comment:
Should probably be `errors.InterfaceError` or a subclass.

Reply:
I'll create a new PR for this - too many of them to fix in a CRF.